README.md

```diff
@@ -62,7 +62,13 @@ Currently the most reliable way of installing Gatekeeper is to build and install
 
 #### Deploying a Release using Prebuilt Image
 
-If you want to deploy a released version of Gatekeeper in your cluster with a prebuilt image, then you can run the following command:
+If you want to deploy a released version of Gatekeeper in your cluster with a prebuilt image, then first copy the following kustomize configuration patch:
+ 
+  ```sh
+  cp config/manager_image_patch_template.yaml config/manager_image_patch.yaml
```

Comment:
`Prebuilt Image` here refers to a published image as part of a release we have previous cut. For example: `quay.io/open-policy-agent/gatekeeper:v3.0.0`

This deployment yaml https://raw.githubusercontent.com/open-policy-agent/gatekeeper/master/deploy/gatekeeper-constraint.yaml should work out of the box, without any modification. If you want to build you own image, then there are two options:
1. Follow [this](https://github.com/open-policy-agent/gatekeeper/blob/master/README.md#deploying-head-using-make) to deploy head using Make
2. Replace the gatekeeper image in gatekeeper-constraint.yaml with your own image.

Reply:
Don't think this step is needed if user is using a prebuilt image.

Reply:
Thanks @ritazh, I wasn't sure since I've only been doing the `make deploy` process. I'll remove this then.